wondrous-app/components/ProjectProfile/types.ts

```diff
@@ -2,11 +2,13 @@ import { NextRouter } from 'next/router';
 import React from 'react';
 import { ENTITIES_TYPES } from 'utils/constants';
 
+export type EntitiesType = typeof ENTITIES_TYPES[keyof typeof ENTITIES_TYPES] | null;
+
 export interface IListItemProps {
   LeftComponent: React.ElementType;
   RightComponent?: React.ElementType;
   data?: object;
-  onClick?: (router: NextRouter, data?) => void;
+  onClick?: (params: { router?: NextRouter; data?; entityLink?: string }) => unknown;
```

Comment:
im not too familiar with this why do we use unknown vs any?

Reply:
actually, I changed this to `void`.

`unknown` is like `any` but you can't do anything with it. https://www.typescriptlang.org/docs/handbook/2/functions.html#unknown